### ISSUE TITLE
[Accessibility][Social Sharing] 3rd party social share accessibility issues

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1804]
**Steps to Reproduce**
1. Press the TAB key to move focus to the Facebook Share button. Observe that no visual indication of focus is present.
2. Close browser, enable NVDA, and reopen the page. 
3. Press TAB key to move focus to Facebook Share button. Observe that it announces "fb:share_button Facebook Social Plugin frame Share 11 button visited link".  (number of shares and visited state may differ.)

**Current Behavior**
* Facebook Share button does not provide visual indicator of keyboard focus, making it difficult for sighted keyboard users to identify where their focus is.
* Facebook Share button iframe title includes text that is not appropriately descriptive. The title is announced to NVDA users as "fb:share_button Facebook Social Plugin".
* Facebook Share button includes invalid nesting of a `button` element within an `a` element, which may prevent some assistive technology from correctly parsing the content and relaying it correctly to the user.

**Expected behavior/code**
All content, including 3rd party content, should be accessible. Facebook share button should provide a visible focus indicator. Facebook share button iframe title should not include the "fb:share_button" text. Interactive elements should not be nested within other interactive elements.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Screen reader users, keyboard-only users

**WCAG SC**
2.4.3 Focus Visible
2.4.6 Headings and Labels
4.1.1 Parsing

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by gabrielwalt** [comment]
The Facebook share code is not generated by the Core Components, but by Facebook.
We'll anyway be dropping the Facebook share features in the new page version: see #1918.
